roles/nodejs/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
 # nodejs version to install
-nodejs_major_version: "4"
+nodejs_major_version: "6"
```

Comment:
We don't handle migrations or other special cases in our Ansible playbooks. This would be utterly complicated to do.

So creating new `ansible-st2` release for this change sounds good. Apart of that, let's merge this after we fully release + announce StackStorm `v2.4.0`.